baiji/util/test_reachability.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+class TestReachability(unittest.TestCase):
+    def test_internet_reachable(self):
+        # We assume these tests run with a reachable Internet. This test is
```

Comment:
Currently, as a group, the "unit tests" in this repo are integration tests which require S3. What you're suggesting is possible.

Will reword the comment to that effect.